seed.js

```diff
@@ -23,16 +23,24 @@ const randoTimeStamp = (maxTime) => {
   return Math.floor(Math.random() * maxTime);
 }
 
-for ( let i = 1; i <= maxComments; i++) {
-  let tempComment = {
-    comment_id: i,
-    user_id: randoUserId(),
-    song_id: randoSongId(),
-    content: lorem.generateSentences(1),
-    time_stamp: randoTimeStamp(maxSongLength)
+const seedComments = async () => {
+  try {
+    await db.dropCollection();
+    for ( let i = 1; i <= maxComments; i++) {
+      let tempComment = {
+        user_id: randoUserId(),
+        song_id: randoSongId(),
+        content: lorem.generateSentences(1),
+        time_stamp: randoTimeStamp(maxSongLength)
+      }
```

Comment:
Are these helper functions elsewhere ie. randoUserID()?

Reply:
I think it's higher up in the file.